scripts/install_third_party_libs.py

```diff
@@ -32,6 +32,8 @@
     ('pyyaml', '5.1.2', os.path.join(TOOLS_DIR, 'pyyaml-5.1.2')),
     ('future', '0.18.2', os.path.join(
         'third_party', 'python_libs')),
+    ('certifi', '2020.12.5', os.path.join(
+        TOOLS_DIR, 'certifi-2020.12.5')),
```

Comment:
Do you think installing it via requirement.txt file would be simple? (Also allows us to easily remove this process of downloading depend. in the future.)

Reply:
Yep, I agree with this, sorry for not mentioning it earlier. Basically, if the code that requires the library is used in backend we should use requirements.txt (if there are no other problems with this approach).

Reply:
I don't think this will work. We run `python -m scripts.install_third_party_libs` right after installing `wheel` in the CI tests, and `install_third_party_libs.py` begins by running `setup.main(args=[])`. In turn, `setup.py` begins by installing node, which uses `url_retrieve`. All this happens before we install the python dependencies in `requirements.txt` (as part of `install_third_party.main(args=[])` in `install_third_party_libs.py`). `url_retrieve` requires certifi, so we need to have certifi installed before we install the packages in `requirements.txt`

Reply:
Okay, makes sense.

Reply:
if python dependencies/code is used for downloading node related stuff then we should download the python dep first but in this case, I think we should not download it through requirement.txt as it will be stored in third-party dir and will get included for deployment which is not needed. (though there's a workaround to add the path in app.yml)